bcc/module.go

```diff
@@ -351,3 +351,24 @@ func (bpf *Module) TableIter() <-chan map[string]interface{} {
 	}()
 	return ch
 }
+
+func (bpf *Module) attachXDP(devName string, fd int) error {
+	devNameCS := C.CString(devName)
+	res, err := C.bpf_attach_xdp(devNameCS, C.int(fd))
+	C.free(unsafe.Pointer(devNameCS))
+
+	if res != 0 || err != nil {
+		return fmt.Errorf("failed to attach BPF xdp to device %v: %v", err, devName)
```

Comment:
Please change the order of variables. You probably want to format `devName` first, then `err`.